api/genomics/src/main/java/org/pmiops/workbench/genomics/RandomizeVcf.java

```diff
@@ -0,0 +1,165 @@
+package org.pmiops.workbench.genomics;
+
+import com.google.common.annotations.VisibleForTesting;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypeBuilder;
+import htsjdk.variant.variantcontext.GenotypesContext;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import htsjdk.variant.variantcontext.writer.VariantContextWriter;
+import htsjdk.variant.vcf.VCFHeader;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.engine.VariantWalker;
+import picard.cmdline.programgroups.VariantManipulationProgramGroup;
+
+@CommandLineProgramProperties(
+    summary = "Generates random variant alleles at the same contigs as an example VCF",
+    oneLineSummary = "Randomizes a VCF",
+    programGroup = VariantManipulationProgramGroup.class)
+public class RandomizeVcf extends VariantWalker {
+  @Argument(
+      doc = "Output vcf name.",
+      fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+      shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME)
+  protected File outputVcf;
+
+  @Argument(doc = "Sample name suffix.", fullName = "SAMPLE_NAME_SUFFIX", shortName = "S")
+  protected String sampleNameSuffix;
+
+  private Random random = new Random();
```

Comment:
I recommend allowing the test to set a fixed seed to make the test more deterministic. See my comment in the test file for how you could plumb that through in a test constructor